libdnf.spec

```diff
@@ -316,12 +316,17 @@ License:        GPLv2+
 Requires:       libdnf%{?_isa} = %{version}-%{release}
 Requires:       libdnf-cli%{?_isa} = %{version}-%{release}
 Requires:       dnf-data
+BuildRequires:  pkgconfig(sdbus-c++)
 
```

Comment:
There might be missing build deps for running the tests:
```
BuildRequires:  python3-devel
BuildRequires: python3dist(dbus-python)
```
I'm not sure if we want to run the dbus tests during build time...